pandas/core/window/rolling.py

```diff
@@ -1610,6 +1610,36 @@ def count(self):
         template_returns,
         create_section_header("See Also"),
         template_see_also[:-1],
+        create_section_header("Examples"),
+        dedent(
+        """
+        Apply function that uses multiple columns as input using
+        method='table'. The apply function multiplies A with B and sums all rows
+        (dot product).
+        
+        >>> df = pd.DataFrame({{'A': range(5), 'B': range(5,0,-1)}})
```

Comment:
I get a `TypeError: unhashable type: 'dict'` when I run this line. When I replace `{{` with `{` and `}}` with `}` it works, though.